travis_install.sh

```diff
@@ -16,9 +16,7 @@ backend() {
 echo "installing $RUNTEST dependencies"
 if [ "$RUNTEST" == "frontend" ]; then
     frontend
-elif [ "$RUNTEST" == "backend" ]; then
     backend
```

Comment:
Did you intend this to be running the backend tests whenever the frontend tests are run?

Reply:
But, if the acceptance tests are run as part of `frontend`, then the dependencies installed by `backend` won't be available yet, will they?

Reply:
Ohhh, I get it now.

OK, last question: Would it be possible add a reduced set of the `backend` dependencies – only the ones we need for acceptance testing – to the `frontend` install and just install those?

Reply:
The requirements would be same. The backend process just runs `requirements/travis.txt`, which just installs the dependencies necessary to run the backend on Travis. The slowness comes from `npm install` and running webpack. I'm working on a separate fix for that.

Reply:
Got it. Makes sense, thanks.